Pinta.Core/ImageFormats/FormatDescriptor.cs

```diff
@@ -78,21 +78,21 @@ public sealed class FormatDescriptor
 		public FormatDescriptor (string displayPrefix, IEnumerable<string> extensions, IEnumerable<string> mimes,
 					 IImageImporter? importer, IImageExporter? exporter)
 		{
-			if (extensions == null || (importer == null && exporter == null))
-				throw new ArgumentNullException ("Format descriptor is initialized incorrectly");
+			if (extensions is null)
```

Comment:
I just noticed this null check is probably unnecessary since we build with nullable types enabled and  `extensions` is not nullable